README.md

```diff
@@ -181,28 +183,33 @@ Note: writes to Azure are not yet supported, please see [the current discussion]
 
 ### Connect with MotherDuck
 
-pg_duckdb also integrates with [MotherDuck][md]. To enable this support you first need to [generate an access token][md-access-token] and then add the following line to your `postgresql.conf` file:
+pg_duckdb also integrates with [MotherDuck][md]. To enable this support you first need to [generate an access token][md-access-token]. Then you can enable MotherDuck by simply using:
 
-```ini
-duckdb.motherduck_token = 'your_access_token'
+```sql
+SELECT duckdb.enable_motherduck('your_access_token');
 ```
 
-NOTE: If you don't want to store the token in your `postgresql.conf`file can also store the token in the `motherduck_token` environment variable and then explicitly enable MotherDuck support in your `postgresql.conf` file:
+NOTE: If you don't want use the token in a query, you may also store the token in the `motherduck_token` environment variable and then explicitly enable MotherDuck support by omitting the token:
 
-```ini
-duckdb.motherduck_enabled = true
+```sql
+SELECT duckdb.enable_motherduck();
 ```
 
-If you installed `pg_duckdb` in a different Postgres database than the default one named `postgres`, then you also need to add the following line to your `postgresql.conf` file:
+The default MotherDuck database will be easiest to use (see below for details), by default this is `my_db`. If you want to specify which MotherDuck database is your default database, then you need to configure MotherDuck using a `SERVER` and a `USER MAPPING` as such:
 
-```ini
-duckdb.motherduck_postgres_database = 'your_database_name'
-```
+```sql
+CREATE SERVER md_server
+TYPE 'motherduck'
+FOREIGN DATA WRAPPER pg_duckdb
+OPTIONS (default_database '<your database>');
 
-The default MotherDuck database will be easiest to use (see below for details), by default this is `my_db`. If you want to specify which MotherDuck database is your default database, then you can also add the following line to your `postgresql.conf` file:
+-- You may use `::FROM_ENV::` to have the token be read from the environment variable
+CREATE USER MAPPING FOR CURRENT_USER SERVER md_server OPTIONS (token '<your token>')
+```
```

Comment:
This can be removed from the README now. I think we should only show the convenience method in the readme. All other stuff is "expert" mode and can be covered in the full docs.

Reply:
Done here [`ac8da79` (#668)](https://github.com/duckdb/pg_duckdb/pull/668/commits/ac8da797b3cf866f7324d0b9b4f918ec90086445)